src/System.CommandLine/Binding/Binding.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.CommandLine.Invocation;
+
+namespace System.CommandLine.Binding
+{
+
+    public delegate void BindingSetter(InvocationContext context, object target, object value);
```

Comment:
Our delegate naming convention (for better or worse) has so far been to make them more method-like, e.g. `Suggest`, `ConvertArgument`, and `ValidateSymbol`. Maybe `SetBinding` and `GetBinding`? Does that change the meaning?